cleanlab/count.py

```diff
@@ -184,10 +184,13 @@ def calibrate_confident_joint(confident_joint, labels, *, multi_label=False) ->
     calibrated_cj : np.ndarray
       An array of shape ``(K, K)`` of type float representing a valid
       estimate of the joint *counts* of noisy and true labels.
+      If multi_label is true, it returns an array of shape ``(K, 2, 2)``,
+      where entry ``(c, j, k)`` in the matrix is the number of examples in a one-vs-rest class confidently counted into the pair of ``(class c, noisy label=j, true label=k)`` classes.
```

Comment:
This notation does not make sense to me. What does ``(class c, noisy label=j, true label=k)`` mean?